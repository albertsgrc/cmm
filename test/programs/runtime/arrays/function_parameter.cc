/*
    description{Checks arrays as function parameters}
*/

int f(int x[13][25]) {
    cout << x << endl;
    return 0;
}

int main() {
    int a[5][13][25];
    f(a[2]);
}
