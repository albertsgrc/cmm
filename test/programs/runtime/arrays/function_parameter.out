0x00000a28
